package.json

```diff
@@ -7,6 +7,20 @@
   "scripts": {
     "build": "yarn workspaces foreach run build",
     "lint": "yarn workspaces foreach run lint",
-    "test": "yarn workspaces foreach run test"
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.2",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-typescript": "^7.16.7",
+    "@testing-library/dom": "^8.11.3",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/react-hooks": "^7.0.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.0",
+    "jest": "^27.5.1",
+    "ts-jest": "^27.1.3",
+    "ts-node": "^10.5.0"
   }
```

Comment:
@vojtechszocs did we want to put our jest stuff in another package? I know you've been trying to keep the root package as clean as possible.

Reply:
Currently we're doing that kind of dependency & config isolation for ESLint stuff via `eslint-plugin-internal` monorepo package.

We should try to have root `package.json` without any dependencies; when a specific monorepo package needs e.g. Jest it should have an explicit dependency in its own package manifest. This aligns with general Yarn 2+ best practices (`yarn run <tool>` generally expects `tool` to be a direct dependency of the given package, for example).

In future, we can create e.g. `testing-internal` monorepo package and have `lib-core` and any other packages to depend on that, while keeping the root `package.json` clean of any dependencies.